src/providers/tfc/workspaceFilters.ts

```diff
@@ -61,14 +69,29 @@ export class ProjectsAPIResource implements APIResource {
       picks.push(...(await this.createProjectItems(this.organizationName, query)));
     } catch (error) {
       let message = 'Failed to fetch projects';
-      if (error instanceof Error) {
-        message = error.message;
+
+      if (error instanceof ZodiosError) {
+        handleZodiosError(error, message, this.outputChannel, this.reporter);
+        return picks;
+      }
+
+      if (axios.isAxiosError(error)) {
```

Comment:
Didn't we use `isErrorFromAlias` here to get better types for `error.response`? That should allow us to get rid of the `?` and the `as ErrorResponse` type cast below.